keps/1093-multi-level-cluster-queues/README.md

```diff
@@ -0,0 +1,151 @@
+# KEP-1093: Multi Level Cluster Queues
+
+<!--
+this is the title of your kep. keep it short, simple, and descriptive. a good
+title can help communicate what the kep is and should be considered as part of
+any review.
+-->
+
+<!--
+a table of contents is helpful for quickly jumping to sections of a kep and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [PlanA](#plana)
+  - [Validation](#validation)
+    - [ClusterQueue](#clusterqueue)
+    - [Hierachy](#hierachy)
+    - [ResourceFlavor](#resourceflavor)
+    - [localqueue](#localqueue)
+  - [Schedule Behavior](#schedule-behavior)
+  - [API](#api)
+- [Implementation](#implementation)
+- [Testing Plan](#testing-plan)
+  - [NonRegression](#nonregression)
+  - [Unit Tests](#unit-tests)
+  - [Integration tests](#integration-tests)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+This proposal allow cluster admins to define a multi-level hierachy for cluster queues. Multi-level of cluster queues allow admins and users to define different dequeue policies for different node. This will allow Kueue to manage more levels of resources.
```

Comment:
Could you elaborate more on use cases that you would like to solve?

Reply:
added a user story